### DISCUSSION TITLE
Inspiration for new tools (it-tools)

### DISCUSSION THREAD

**Discussion by george-gca** [DevToys-app/DevToys#1261] [Ideas]
It would be nice to see many of the solutions already supported in [it-tools](https://it-tools.tech/) supported here. I know that more than half already are.

I know it is a different proposal, since it is web technology and not exactly focused on offline usage, but it is a similar application with lots of different useful tools inside.

**Comment by rgueldenpfennig** [comment]
Which of those would be your favourite tool to get implemented in DevToys?

**Reply by george-gca** [reply]
I believe the most useful would be:
- Password strength analyser
- Converters
  - TOML
  - XML
  - Color converter (RGB, HEX, HSV, ...)
- JSON minifier
- Chmod calculator
- Docker run to Docker compose converter
- Lorem ipsum generator
- Emoji picker

**Reply by veler** [reply]
For your information Json minifier and lorem ipsum generator are already available in DevToys

**Reply by george-gca** [reply]
I said json but I was thinking javascript lol. And I just missed the lorem ipsum, sorry about that.
